assets/src/editor/events/ticket-assignments-manager/filters/date-filters.js

```diff
@@ -14,22 +22,25 @@ const { FormColumn } = twoColumnAdminFormLayout;
  * @function
  * @param {boolean} showArchivedDates
  * @param {boolean} showExpiredDates
- * @param {Function} toggleArchivedDates
- * @param {Function} toggleExpiredDates
- * @param {boolean} showDateFilters
- * @param {boolean} showTicketFilters
+ * @param {number} dateFiltersOffset
+ * @param {Function} setFilter
  * @return {Object} rendered date filter toggles
  */
 const DateFilters = ( {
 	showArchivedDates,
 	showExpiredDates,
-	toggleArchivedDates,
-	toggleExpiredDates,
-	showDateFilters,
-	showTicketFilters,
+	dateFiltersOffset,
+	setFilter,
 } ) => {
-	const dateFiltersOffset = showDateFilters && showTicketFilters ? 2 : 7;
-	return showDateFilters ? (
+	const toggleArchivedDates = useCallback(
+		() => setFilter( showArchivedDatesAction ),
+		[]
+	);
+	const toggleExpiredDates = useCallback(
+		() => setFilter( showExpiredDatesAction ),
+		[]
+	);
```

Comment:
This should have `setFilter` and `showExpiredDatesAction` as dependencies.

Reply:
see response above